src/GAP/iso_oscar_gap.jl

```diff
@@ -55,12 +55,18 @@ function _iso_oscar_gap(RO::Union{Nemo.zzModRing, Nemo.ZZModRing})
    return MapFromFunc(RO, RG, f, finv)
 end
 
-# Assume that `FO` and `FG` are finite fields of the same order
-# in Oscar and GAP, respectively.
-function _iso_oscar_gap_field_finite_functions(FO::Union{Nemo.fpField, Nemo.FpField}, FG::GAP.GapObj)
+_ffe_to_int(a::FqFieldElem) = Nemo._coeff(a, 0)
+_ffe_to_int(a::FqPolyRepFieldElem) = coeff(a, 0)
+_ffe_to_int(a::fqPolyRepFieldElem) = coeff(a, 0)
+_ffe_to_int(a::Union{fpFieldElem,FpFieldElem}) = lift(a)
+
+function _make_prime_field_functions(FO, FG)
    e = GAPWrap.One(FG)
 
-   f(x) = GAP.Obj(lift(x))*e
+   f = function(x)
+     y = GAP.julia_to_gap(_ffe_to_int(x))::GapInt
```

Comment:
@ThomasBreuer For now I removed the `ZZ(...)` from the one `_ffe_to_int` method and just call `GAP.julia_to_gap` here, that should work fine for now. On the long run my plan is to use `GAP.GapInt` instead, see https://github.com/oscar-system/GAP.jl/pull/984